lib/pbench/test/unit/agent/test_tool_group.py

```diff
@@ -1,9 +1,10 @@
 import os
 from pathlib import Path
+from unittest import mock
```

Comment:
Why are we adding `unittest.mock` instead of using PyTest's `monkeypatch` fixture?

(`unittest` looks very cool, but I'm concerned about having more than one way to do things sewing confusion.)

Reply:
For some reason I keep forgetting about `monkeypatch`.  I'll see about re-working that.

Reply:
This means that you are a Good Programmer -- it means that you're writing new code instead of just stealing old code.

("Good Programmers write Good Code; Great Programmers steal Good Code." 🤣)